pkg/kubelet/dockershim/docker_service.go

```diff
@@ -139,17 +139,15 @@ type DockerService interface {
 	internalApi.ImageManagerService
 	DockerLegacyService
 	Start() error
+	// For serving streaming calls.
+	http.Handler
 }
 
 // DockerLegacyService is an interface that embeds all legacy methods for
 // backward compatibility.
 type DockerLegacyService interface {
```

Comment:
Rebase and remove the `DockerLegacyService` completely since `GetContainerLogs` has been removed.